imdb/SearchResults.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import sys
+import traceback
+
+import requests
+
+from bs4 import BeautifulSoup
+
+
+def get_search_movies(movie):
+
+    url = "http://www.imdb.com/find?s=tt&q=" + movie
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        logging.info(e)
+        traceback.print_exc(e)
+        json_object = {}
+        json_object['error'] = "Service Unavailable"
+        tup = (True, 503, json_object)
+        return tup
+
+    soup = BeautifulSoup(r.content, 'html.parser')
+    movies = soup.find_all("td", {"class": "result_text"})
+
+    json_array = []
+
+    if len(movies) == 0:
+        json_object = {}
+        json_object['error'] = "no results found"
+        tup = (True, 404, json_object)
+        return tup
+
+    for item in movies:
+        if item is not None:
+            title = item.get_text().strip()
+            url = item.find("a").get("href")
+            movie_id = url[7:16]
+
+        url = 'http://imdb.com' +url
+        json_object= {}
+        json_object['title'] = title
+        json_object['url'] = url
+        json_object['id'] = movie_id
+        json_array.append(json_object)
```

Comment:
append is used to append items  to the array of the json_object after each iteration and increase the ijndex of the array

extent or extend??